examples/with-federated-authentication/src/pages/index.tsx

```diff
@@ -0,0 +1,162 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+import { getWebAuthnAttestation, TurnkeyApi } from "@turnkey/http";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { buffer } from "stream/consumers";
+import { useState } from "react";
+
+type subOrgFormData = {
+  orgName: string;
+};
+
+type privateKeyFormData = {
+  privateKeyName: string;
+};
+
+const es256 = -7;
+const publicKey = "public-key";
```

Comment:
Likewise. A link to the spec + a mention of the fact that we're always creating public key credential would be nice. The reason why it's good to be verbose here is that this example is meant to be copy/pasted; the more explicit we are on the "why" the better.